<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWnd</class>
 <widget class="QMainWindow" name="CMainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote chassis control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/ctrl.png</normaloff>:/images/ctrl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
   <addaction name="actionMoto"/>
   <addaction name="actionCtrl"/>
   <addaction name="actionSudoctrl"/>
   <addaction name="separator"/>
   <addaction name="actionChat"/>
   <addaction name="actionClientlist"/>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/network.png</normaloff>:/images/network.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to host</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/chat.png</normaloff>:/images/chat.png</iconset>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
   <property name="toolTip">
    <string>Chat window</string>
   </property>
  </action>
  <action name="actionCtrl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/ctrl.png</normaloff>:/images/ctrl.png</iconset>
   </property>
   <property name="text">
    <string>Ctrl</string>
   </property>
   <property name="toolTip">
    <string>Query hardware control</string>
   </property>
  </action>
  <action name="actionSudoctrl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/sudoctrl.png</normaloff>:/images/sudoctrl.png</iconset>
   </property>
   <property name="text">
    <string>Sudo ctrl</string>
   </property>
   <property name="toolTip">
    <string>Query hardware control with extra privilegies (password required).</string>
   </property>
  </action>
  <action name="actionMoto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/motor.png</normaloff>:/images/motor.png</iconset>
   </property>
   <property name="text">
    <string>Moto</string>
   </property>
   <property name="toolTip">
    <string>Motors control window</string>
   </property>
  </action>
  <action name="actionClientlist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/userlist.png</normaloff>:/images/userlist.png</iconset>
   </property>
   <property name="text">
    <string>Client list</string>
   </property>
   <property name="toolTip">
    <string>Client list window</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
